src/lib/css-tag.ts

```diff
@@ -37,6 +44,10 @@ export class CSSResult {
   }
 }
 
+export const unsafeCss = (value: any) => {
```

Comment:
Prefer `unknown` to `any` here (and generally).

Reply:
Done.